service_container/calls.rst

```diff
@@ -144,3 +141,21 @@ The configuration to tell the container it should do so would be like:
 
         $container->register(MessageGenerator::class)
             ->addMethodCall('withLogger', [new Reference('logger')], true);
+
+If autowire is enabled, you can also use annotations; with the previous exemple it would be:
```

Comment:
We use "RST magic highlighting" for PHP code. So, instead of `.. code-block:: php`, you need to remove that directive and add two colons `::` in the last paragraph before the code you want to highlight with PHP. So:

```rst
If autowire is enabled, you can also use annotations; with the previous example it would be::
```